### ISSUE TITLE
navigationCurrentItemClass is not updated when navigation uses custom append

### ISSUE THREAD

**Issue by Megagator** [jedrzejchalubek/Glide.js#62]
When the navigation element is appended to a custom HTML class:

```
<div class=".className"></div>

<script>
	$('.slider').glide({
		navigation: '.className'
		});
</script>
```

the navigation items do not pick up the ```.slider-nav__item--current``` class when it advances automatically to the next slide, or if the nav item is clicked to reach a new slide. 
This is can be replicated in the example.html in the source.

**Comment by FrancoTanzarella** [comment]
Any progress or workaround on this?

**Comment by jedrzejchalubek** [comment]
Glide has been released in version v2.0.0. If the problem remains, please set up a new thread.
